src/extension/options-page/PersonaCard.tsx

```diff
@@ -0,0 +1,224 @@
+                    visibility: 'visible',
+                },
+            },
+        },
+    }),
+)
+
+const Link = React.forwardRef<HTMLAnchorElement, RouterLinkProps>((props, ref) => (
+    <RouterLink innerRef={ref} {...props} />
+))
+
+export default function PersonaCard({ identity }: Props) {
+    const classes = useStyles()
+    const color = useColorProvider()
+
+    const [provePost, setProvePost] = useState<string>('')
+
+    useMemo(() => {
+        Services.Crypto.getMyProveBio(identity.identifier).then(p => setProvePost(p || ''))
+    }, [identity.identifier])
+
+    const [friendlyName, setFriendlyName] = useState<string>('')
+
+    useMemo(() => {
+        const ui = [...definedSocialNetworkUIs].find(i => i.networkIdentifier === identity.identifier.network)
+        setFriendlyName(ui ? ui.friendlyName : `Unknown(${identity.identifier.network})`)
+    }, [identity.identifier.network])
+
+    const { enqueueSnackbar, closeSnackbar } = useSnackbar()
+
+    const copyPublicKey = () => {
+        navigator.clipboard
+            .writeText(provePost)
+            .then(() => {
+                enqueueSnackbar('Copied.', { variant: 'success', autoHideDuration: 1000 })
+            })
+            .catch(e => {
+                enqueueSnackbar('Copy Failed.' + provePost, { variant: 'error' })
+            })
+    }
+
+    const undoDeleteIdentity = (restore: BackupJSONFileLatest) => {
+        const undo = (key: any) => {
+            Services.People.restoreBackup(restore)
+            closeSnackbar(key)
+        }
+        return (key: any) => (
+            <Button color="secondary" onClick={() => undo(key)}>
+                Undo
+            </Button>
+        )
+    }
+
+    const deleteIdentity = async () => {
+        const backup = await Services.Welcome.backupMyKeyPair(identity.identifier, {
+            download: false,
+            onlyBackupWhoAmI: true,
+        })
+        Services.People.removeMyIdentity(identity.identifier).then(() => {
+            enqueueSnackbar('Deleted.', { variant: 'default', action: undoDeleteIdentity(backup) })
+        })
+    }
+
+    const [rename, setRename] = useState(false)
+
+    const renameIdentity = (event: React.FocusEvent<HTMLSpanElement> | React.KeyboardEvent<HTMLSpanElement>) => {
+        event.preventDefault()
+        Services.People.updatePersonInfo(identity.identifier, { nickname: event.currentTarget.innerText }).then(() => {
+            enqueueSnackbar('Done.', { variant: 'success', autoHideDuration: 1000 })
+            setRename(false)
+        })
+    }
+
+    const titleRef = useRef<HTMLSpanElement | null>(null)
+
+    return (
+        <Card className={classes.card} raised elevation={3}>
+            <CardContent>
+                <Typography className={classes.header} variant="h5" component="h2" gutterBottom>
+                    {
+                        <>
+                            <span
+                                suppressContentEditableWarning
```

Comment:
use Input instead

Reply:
don't know how to style Input the same as span

Reply:
https://github.com/DimensionDev/Maskbook/issues/355